flag.go

```diff
@@ -881,7 +887,29 @@ func (f *FlagSet) parseArgs(args []string) (err error) {
 func (f *FlagSet) Parse(arguments []string) error {
 	f.parsed = true
 	f.args = make([]string, 0, len(arguments))
-	err := f.parseArgs(arguments)
+	err := f.parseArgs(arguments, nil)
```

Comment:
@eparis I change it on your suggestion. If people didn't give the value when the flag used, will return an empty string for value.  They can use Lookup/LookupByShorthand to get the value. Is it OK?